### PR TITLE
PrBoom-plus: add icon and menu entry

### PR DESCRIPTION
**Pull request by Peppersawce**
An update to the prboom-plus recipe so the game gets its own vector icon and menu entry.
Keep in mind that the menu entry will fail silently if prboom-plus cannot find any valid iwad file in its search directories.

### CODE REVIEW

File: games-fps/prboom-plus/prboom_plus-2.6.66.recipe [hunk 1]
```diff
@@ -87,4 +88,18 @@ INSTALL()
...
+
+	# Generate the rdef
+	local MAJOR="`echo "$portVersion" | cut -b1`"
+	local MIDDLE="`echo "$portVersion" | cut -b3`"
+	local MINOR="`echo "$portVersion" | cut -b5,6`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@VERSION@|$VERSION|" \
```
Comment (jmairboeck) [inline games-fps/prboom-plus/prboom_plus-2.6.66.recipe:100]:
```suggestion
```

Comment (Peppersawce) [reply games-fps/prboom-plus/prboom_plus-2.6.66.recipe:100]: Ah, copy-paste failed me yet again!

**Comment by jmairboeck** [comment]
It is installing the binary to `$prefix/bin/x86`, that means there is something wrong with the CMake setting for the bin dir. The variable should probably be `CMAKE_INSTALL_BINDIR` instead of `CMAKE_INSTALL_BIN` in line 80.

**Comment by Peppersawce** [comment]
> It is installing the binary to `$prefix/bin/x86`, that means there is something wrong with the CMake setting for the bin dir. The variable should probably be `CMAKE_INSTALL_BINDIR` instead of `CMAKE_INSTALL_BIN` in line 80.
I'll check the recipe and fix it later today, thank you guys for reporting.

**Pull request merged**